asreview/webapp/src/SettingsHooks.js

```diff
@@ -77,4 +77,58 @@ const useUndoEnabled = () => {
 };
 
 
-export { useDarkMode, useTextSize, useUndoEnabled };
+const useKeyPress = (targetKey) => {
+
+  const [keyPressed, setKeyPressed] = useState(false);
+  
+  useEffect(() => {
+
+    const downHandler = ({ key }) => {
+      if (key.toLowerCase() === targetKey) {
+        setKeyPressed(true);
+      }
+    };
+
+    const upHandler = ({ key }) => {
+      if (key.toLowerCase() === targetKey) {
+        setKeyPressed(false);
+      }
+    };
+
+    // Add event listeners
+    window.addEventListener('keydown', downHandler);
+    window.addEventListener('keyup', upHandler);
+    // Remove event listeners on cleanup
+    return () => {
+      window.removeEventListener('keydown', downHandler);
+      window.removeEventListener('keyup', upHandler);
+    };
+
+  }, [targetKey]);
+
+  return keyPressed;
+};
```

Comment:
Does it make sense to put this one over here? It looks like something we can move to a standalone file in the subfolder with the review components. What do you think @terrymyc?

Reply:
It was a standalone file, but I had no idea which folder I should put it in since it is a hook. Do you have any advice?

Reply:
Shall we make a `hooks` folder in `src`? I like the idea of making this one into a hook btw. 

There is also another hook for the Help dialog. I think we can move that one there as well (I will do this one).